crates/sel4-hal-adapters/Cargo.nix

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2023, Colias Group, LLC
+# Copyright 2023, Galois, Inc.
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+{ mk, versions, localCrates, smoltcpWith, serdeWith, authors }:
+
+mk {
```

Comment:
If you replace these lines with

```
{ mk, mkDefaultFrontmatterWithReuseArgs, defaultReuseFrontmatterArgs, versions, localCrates, smoltcpWith, serdeWith, authors }:

mk {
  nix.frontmatter = mkDefaultFrontmatterWithReuseArgs (defaultReuseFrontmatterArgs // {
    copyrightLines = defaultReuseFrontmatterArgs.copyrightLines ++ [
      "Copyright 2023, Galois, Inc."
    ];
  });
```

then the copyright lines in the adjacent generated Cargo.toml file will match those in this file.

Not sure if anyone cares about the copyright lines in the generated Cargo.toml files, though.

Reply:
Or, after rebasing on top of https://github.com/seL4/rust-sel4/pull/49,

```
{ mk, overrideDefaultFrontmatter, versions, localCrates, smoltcpWith, serdeWith, authors }:

mk {
  nix.frontmatter = overrideDefaultFrontmatter {
    extraCopyrightLines = [
      "Copyright 2023, Galois, Inc."
    ];
  };
```

Reply:
Again, only if you care about copyright info in the generated Cargo.toml file.